Sadly cmake is broken. If it sees this reference and ansidecl is present, it will add a 
dependency upon it, even if HAVE_ANSIDEC_H is never set.

The easiest solution is to remove these lines, otherwise recipes like libzypp can have a 
dependency on the ansidecl.h header via cmake. This can lead to odd results if the 
header is removed (clean binutils) and then the code is recompiled.

RP 2012/7/10

Upstream-Status: Inappropriate [its really a cmake bug]

diff --git a/configure.ac b/configure.ac
index 0260281..fdb58e9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -484 +483,0 @@ AC_CHECK_HEADERS([time.h])
-AC_CHECK_HEADERS([ansidecl.h])
diff --git a/include/libxml/xmlversion.h.in b/include/libxml/xmlversion.h.in
index b173be9..d10f975 100644
--- a/include/libxml/xmlversion.h.in
+++ b/include/libxml/xmlversion.h.in
@@ -413,3 +412,0 @@ XMLPUBFUN void XMLCALL xmlCheckVersion(int version);
-#ifdef HAVE_ANSIDECL_H
-#include <ansidecl.h>
-#endif
